src/bucklescript/graphql_ppx.re

```diff
@@ -308,6 +309,13 @@ let args = [
     ),
     "A leaner parse function (experimental)",
   ),
+  (
+    "-records",
```

Comment:
I would prefer this to be per query configuration rather than global. I would like to introduce it to new queries or one by one but this option forces me to migrate my whole codebase at once.

Reply:
Agree! I This was also my thinking but I also wanted to have a global flag to make it the default.